### DISCUSSION TITLE
Small bug in the example of 'on_load'

### DISCUSSION THREAD

**Discussion by nphaibk** [r0x0r/pywebview#1124] [General]
Just to inform that there is a bug in the example for event `on_load()`, code below. To unsubscribe the event listener, the command should be `webview.windows[0].events.loaded -= on_load`. Without `events`, error will be raised.

`def on_loaded():

    print('DOM is ready')
    # unsubscribe event listener
    webview.windows[0].loaded -= on_loaded
    webview.windows[0].load_url('https://pywebview.flowrl.com/hello')
`

**Comment by r0x0r** [comment]
fixed
